docs/library/overlay/alertdialog.md

```diff
@@ -10,6 +10,52 @@ components:
 
 only_low_level:
     - True
+
+AlertDialogRoot: |
+    lambda **props: rx.radix.themes.alertdialog_root(
+        rx.radix.themes.alertdialog_trigger(
+            rx.radix.themes.button("Revoke access"),
+        ),
+        rx.radix.themes.alertdialog_content(
+            rx.radix.themes.alertdialog_title("Revoke access"),
+            rx.radix.themes.alertdialog_description(
+                "Are you sure? This application will no longer be accessible and any existing sessions will be expired.",
+            ),
+            rx.radix.themes.flex(
```

Comment:
Perhaps the whole thing should be in a flex. there's not enough padding between the text and the buttons